pkgs/top-level/all-packages.nix

```diff
@@ -3189,8 +3189,6 @@ with pkgs;
 
   aptly = callPackage ../tools/misc/aptly { };
 
-  ArchiSteamFarm = callPackage ../applications/misc/ArchiSteamFarm { };
```

Comment:
This will break the update script https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/ArchiSteamFarm/update.sh